### PR TITLE
Add OpenCV headers to include dirs and linked libraries

### PR DESCRIPTION
**Pull request by willcbaker**
wasn't finding opencv header files during build

### CODE REVIEW

**Comment by sebastianscherer** [comment]
Good catch, thank you! I guess we did not run into this on our machines since the includes were [added automatically for some versions](https://github.com/opencv/opencv/issues/4447).

Could you amend your commit to include a sign-off as mentioned [here](https://github.com/uuvsimulator/uuv_simulator/blob/master/CONTRIBUTING.md)?

File: uuv_sensor_plugins/uuv_sensor_plugins/CMakeLists.txt [hunk 1]
```diff
@@ -92,7 +94,7 @@ add_library(uuv_gazebo_camera SHARED
   src/SensorPlugin.cc
   src/UnderwaterCameraPlugin.cc
 )
-target_link_libraries(uuv_gazebo_camera ${common_libraries}
+target_link_libraries(uuv_gazebo_camera ${common_libraries} ${OpenCV_INCLUDE_DIRS}
```
Comment (sebastianscherer) [inline uuv_sensor_plugins/uuv_sensor_plugins/CMakeLists.txt]: I think this should be ${OpenCV_LIBS} or ${OpenCV_LIBRARIES}

**Comment by willcbaker** [comment]
are these fixes complete?

**Comment by musamarcusso** [comment]
Hi @willcbaker, 

sorry it took so long to respond. I had to check with the regulations here at the company with some colleagues to see if I could accept the pull request. The issue is that your first commit doesn't have the signed off message. Could you amend you first commit to include the signed off message like the second one? 
I know that it sounds like a lot of trouble for a bug fix, but unfortunately I really can't do anything against the company regulations here.
Another (maybe quicker) fix would be to make a new branch and new pull request with the signed off flag to all the commit messages. Or I can add your fixes myself. Let me know what you think is best.
Sorry for the trouble, it is only that we have internal company guidelines to follow. Don't take this as a discouragement to contribute, we really appreciate all contributions :)

Best,

Musa

**Comment by willcbaker** [comment]
@musamarcusso I understand.
I was having issues rebasing the commits on my machine, should be good to go now?

**Comment by musamarcusso** [comment]
Hi @willcbaker, I am restructuring some of the docs for the repository here. Since you contributed to the package, I would say you could also be added to the [NOTICE](https://github.com/uuvsimulator/uuv_simulator/blob/master/NOTICE) file. Would you be OK with that?

**Comment by willcbaker** [comment]
Sure!

**Comment by musamarcusso** [comment]
Great. Could you send me how you would like to listed in the NOTICE file? Check the [current notice](https://github.com/uuvsimulator/uuv_simulator/blob/master/NOTICE#L25) file and the [contribution guidelines](https://github.com/uuvsimulator/uuv_simulator/blob/master/CONTRIBUTING.md) for examples on how it should look like. As soon as you send me this, I can add it to the NOTICE file.

**Comment by willcbaker** [comment]
William Baker <willcbaker at gmail dot com>

**Pull request merged**